nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/new-parameter-context-dialog.jsp

```diff
@@ -117,6 +117,10 @@
             <div class="clear"></div>
         </div>
     </div>
+    <div id="parameter-context-updating-status">
+        <div class='parameter-context-step ajax-loading'></div>
+        <div class='status-message'>Updating parameter context</div>
```

Comment:
@scottyaslan  The problem is that this status message container is not within the same container as the dialog buttons since they are rendered by the dialog component. This is placed at the bottom with absolute positioning and doesn't know how much space it has to draw itself relative to the buttons.

It is a one-off scenario that no other dialogs currently need. Is it worth plumbing something like that through to support status messages so we could "ellipsify" the message as the dialog gets smaller? It is only an issue if the browser window is less than 370px currently. NiFi really isn't usable at that width anyhow.